### ISSUE TITLE
'tickFormat' property should affect only Axis for which it be declared

### ISSUE THREAD

**Issue by alexwizp** [elastic/elastic-charts#185]
**Version (please complete the following information):**
3.11.2

**Describe the bug**
 'tickFormat' property should affect only Axis for which it be declared

**To Reproduce**

1. Create Chart with two Y-axis values
2. Set 'tickFormat' format for **first** Axis e.g` {{value}}F`
4. See result

**Expected behavior**
I expect to see '1F' for first axis and 1 for second

**Screenshots**
![image](https://user-images.githubusercontent.com/20072247/56585409-6d2c1600-65e6-11e9-900d-2f831f062560.png)

**Checklist**
- [x] every related Kibana issue is listed under `Kibana Cross Issues` list
- [x] `kibana cross issue` tag is associated to the issue if any kibana cross issue is present

**Comment by markov00** [comment]
@alexwizp for this case let me summarize the possibility:

1) if you want to visualize multiple series of the same data type, bytes, the same currency etc, you can just add a single Axis with a single `tickFormat`.

2) if you want to add multiple series from different data types, like you want to add one series with currency and another with bytes, you should specify 2 Axis components. You should also specify two different groupIds one for the bytes and one for the currency, because you are displaying two different values on two different domains (bytes and currency). 

2.5) if you want to have the same scale, for any specific reason, actually you can manually compute the min and max domain of the two series and apply the same domain to each series. You can then hide one or the other axis to have only one axis displayed.
Basically it works in the same way works for https://github.com/elastic/elastic-charts/issues/169#issuecomment-483656765

**Comment by markov00** [comment]
We can add a basic formatter for each individual series, that will override, on the tooltip and on the legend the formatter of the axis if you think it's necessary.
